Main.java

```diff
@@ -10,6 +12,25 @@
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
+     Scanner input = new Scanner(System.in); 
+
+    // Ask the user to input two numbers to divide 
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+
+    //Take the first number 
+    int X = input.nextInt(); 
+
+    //Take the second number 
+    int Y = input.nextInt(); 
+
+    //divide the numbers  
+    int A = X / Y; 
+
+    // tell the user the answer 
+    System.out.println(X + "/" + Y +  "is"  + A +  "with a remainder of" +  X%Y);
```

Comment:
Good work determining the math and commenting the code. Just remember that variable names should always begin with a lowercase letter and describe what they are storing or what they are being used for. This means that X, Y, and A are probably bad choices for variable names. Instead, something like number1, number2 and quotient should be used.

Also, try to avoid doing actual calculations inside of a print statement. Instead, do the calculations inside of a variable and then use that variable to print to the screen.